### PR TITLE
Updated README:

### PR DESCRIPTION
**Pull request by nbriannl**
> Guys, the original README file in your repo (which doubles as the landing page of your project website) is written to introduce the project as a learning/teaching resource. You are recommended to rephrase it to read like the home page of a real product (not a school project). e.g. remove references to AB3, Learning Outcomes etc. mention target users, add a marketing blurb etc.
_- By Damith_

### CODE REVIEW

**Review by zacharytang — APPROVED** [review]

File: README.adoc [hunk 1]
```diff
@@ -15,26 +15,27 @@ ifndef::env-github[]
...
-* For developers, this application is *written in OOP fashion*. The code base is *reasonably well-written* and *significantly big* (around 6 KLoC).
-* Source code is based on the  https://github.com/se-edu/addressbook-level4[AddressBook-Level4] project created by SE-EDU initiative.
-* This application is an enhanced version of AddressBook-Level 4
-** More Commands, and a more intuitive Command Line syntax.
-** More attractive UI
-** And many other features targeting the *university student*.
+__Unifying your University Contact Information, and more. For You.__
+
+* This is a desktop Address Book application for University students, particularly those from NUS (National University of Singapore).
+* Manage the many people you meet in University, keep track of your friends' Birthdays, view your friend's timetables, plus much more!
```
Comment (April0616) [inline README.adoc:21]: and matriculation numbers

**Review by April0616 — APPROVED** [review]
